azdev/operations/performance.py

```diff
@@ -15,7 +15,7 @@
 logger = get_logger(__name__)
 
 TOTAL = 'ALL'
-TOTAL_THRESHOLD = 300
+TOTAL_THRESHOLD = 400   # temporary increase to 400 for CDN
```

Comment:
I don't think changing this value is enough. We also need to tweak `THRESHOLDS`. 

But let me investigate why CDN is slow first.